### PR TITLE
Fun-O-Meter dashboard improvements

### PR DESCRIPTION
**Pull request by ashercodeorg**
Links to script and script-level views. Increases graph size. Makes graph selectable.

### CODE REVIEW

File: dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .by_day
   %h3
     Percentages by Day
-  #chart_by_day
+  %span
+    #chart_by_day
```
Comment (Hamms) [inline dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml]: Div inside of a span? Why?

**Comment by Hamms** [comment]
One question, otherwise LGTM!

File: dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml]: Done.

File: dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 .by_day
   %h3
     Percentages by Day
-  #chart_by_day
+  %div
+    #chart_by_day
```
Comment (Hamms) [inline dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml]: what was the motivation to put `#chart_by_day` inside a container in the first place?

Comment (ashercodeorg) [inline dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml]: The motivation was misplaced. Let me play around a bit.

**Pull request merged**